### DISCUSSION TITLE
Why installer need admin right?

### DISCUSSION THREAD

**Discussion by jean-io** [keepassxreboot/keepassxc#10571] [Q&A] [answered]
Hi, I would like to use KeepPassXC on a computer on which I am not admin, but I cannot install it since the installer need admin rights. There is no explanation in installer on why this is necessary and there is no option to do a "local" installation.

Even with a local path of installation and without the need of auto-start, privileges are required... **why**?

![image](https://github.com/keepassxreboot/keepassxc/assets/4070933/a42598a4-a9f9-43af-af91-478d0abb212e)

**Comment by droidmonkey** [comment]
That is interesting, the installer shouldn't ask for escalation unless it needs it.

However, you can use the portable download (.zip) to accomplish your task.

**Reply by jean-io** [reply, marked as answer]
I did not know that ZIP in release page on GitHub is actually the portable version of KeepassXC... this is good to know! Thank. A proper installation would be better. Is this an issue? should I transform this topic into an issue?

**Reply by droidmonkey** [reply]
I created an issue for this: https://github.com/keepassxreboot/keepassxc/issues/10572
